From: Neil Williams <neil.williams@linaro.org>
Date: Fri, 5 Oct 2018 10:07:32 +0100
Subject: Remove setuptools_scm_git_archive from setup.py

This may be a bug in setuptools_scm_git_archive but
setuptools is unable to locate it even though a subsequent
import works.
Remove enum34 from the requires to convert to Python3.
---
 setup.py | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/setup.py b/setup.py
index 742d225..3baac11 100644
--- a/setup.py
+++ b/setup.py
@@ -21,7 +21,6 @@ import sys
 install_requires = [
     'intelhex',
     'six',
-    'enum34',
     'future',
     'websocket-client',
     'intervaltree',
@@ -47,7 +46,7 @@ setup(
         'local_scheme': 'dirty-tag',
         'write_to': 'pyocd/_version.py'
     },
-    setup_requires=['setuptools_scm!=1.5.3,!=1.5.4', 'setuptools_scm_git_archive'],
+    setup_requires=['setuptools_scm!=1.5.3,!=1.5.4'],
     description="CMSIS-DAP debugger for Python",
     long_description=open('README.rst', 'r').read(),
     author="Chris Reed, Martin Kojtal, Russ Butler",
